go.mod

```diff
@@ -1 +1,3 @@
-module github.com/mitchellh/mapstructure
```

Comment:
Are we planning to make a pull request up to that repo? Should we make a separate PR with this change so we can at least attempt a PR into that repo?

Reply:
The PR that implements this change, https://github.com/mitchellh/mapstructure/pull/155, has been open since February.